packages/frontend/src/utils/colorUtils.ts

```diff
@@ -52,14 +52,13 @@ export const getColorFromRange = (
     },
 ): string | undefined => {
     const colors = getColorRange(config.color);
+    if (!colors) return undefined;
```

Comment:
has the logic here changed or just tidied up?

Reply:
tidy up